perfkitbenchmarker/openstack/os_virtual_machine.py

```diff
@@ -31,7 +31,7 @@
 flags.DEFINE_boolean('openstack_boot_from_volume', False,
                      'Boot from volume instead of an image')
 
-flags.DEFINE_integer('openstack_volume_size', 20,
+flags.DEFINE_integer('openstack_volume_size', -1,
```

Comment:
DEFINE_integer should support None as the default value if you want to use that. I recommend explaining the what the None/-1 means in the flag description string.